README.md

```diff
@@ -174,23 +173,37 @@ If you have v1.0.1 instealled then your libyear when checking on May 1, 2019 is
 
 ### Upgrades Available
 
-Is simply the number of dependencies in your project that are not using the latest version at the time of the check.  For example, if you have 5 dependencies and 3 of them are not using the latest version then you the update count is 3.
+Is simply the number of dependencies in your project that are not using the latest version at the time of the check. For example, if you have 5 dependencies and 3 of them are not using the latest version then you the update count is 3.
 
 ### Skipped
 
-Simply the number of dependencies Freshli could not calculate the libyear for on the given date.  Could be because a package has been removed in the package manager or similar issue.  Could also be a timeout error from the package manager or a bug in Freshli.
+Simply the number of dependencies Freshli could not calculate the libyear for on the given date. Could be because a package has been removed in the package manager or similar issue. Could also be a timeout error from the package manager or a bug in Freshli.
 
 ## Culture and Language Support
 
 The headings for column output are localized such that the culture settings of the user's computer are used. (This is found in the CurrentUICulture). Currently there are English and Spanish translations with English being the default.
 
 Data (such as dates and numeric formatting) are NOT localized. Dates and numeric formats use the CurrentCulture which is explicitly set to the [invariant culture](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-5.0).
 
-We are not sure how to handle documentation, such as this ReadMe, in different languages.  If you have any suggestions or would like to help with translations please let us know using the contact information in the [Contributing](#contributing) section.
+We are not sure how to handle documentation, such as this ReadMe, in different languages. If you have any suggestions or would like to help with translations please let us know using the contact information in the [Contributing](#contributing) section.
+
+## Logging
+
+By default all logs are set to WARN level and sent to the console output.
+
+Log levels can be adjusted by using the `--logLevel <level>` option when running the application. The level can be any level that is supported by NLog:
+
+-   Trace
+-   Debug
+-   Info
+-   Warn
+-   Error
+-   Fatal
+
+Logs can be redirected to a file instead by using the `--logFile <file_path_and_name>` option when running the application.
```

Comment:
Can you add an example of a command line that uses the `logLevel` and `logFile` options?

Reply:
Issue #214 has been created to track the need to update the README with this information.